app/src/main/java/org/cru/godtools/activity/BasePlatformActivity.kt

```diff
@@ -80,6 +80,7 @@ abstract class BasePlatformActivity : BaseDesignActivity(), NavigationView.OnNav
     private var loginItem: MenuItem? = null
     private var signupItem: MenuItem? = null
     private var logoutItem: MenuItem? = null
+    private var tutorialItem: MenuItem? = null
```

Comment:
not necessary if we just set the visibility inline when setting up the nav drawer